### PR TITLE
Access templates implementation

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-access/0.1/amp-access.js [hunk 1]
```diff
@@ -259,9 +266,46 @@ export class AccessService {
...
 
-    // TODO(dvoytenko): support templates
+    /**
+     * Render templates.
+     * @type {!Array<!{template: !Element, rendered: !Promise<!Element>,
+     *    prev: ?Element}>}
+     */
+    const templates = [];
+    if (on) {
+      const templateElements = element.querySelectorAll(
```
Comment (cramforce) [inline extensions/amp-access/0.1/amp-access.js]: factor into method?

**Comment by cramforce** [comment]
LGTM

File: extensions/amp-access/0.1/amp-access.js (hunk 1 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access.js]: Done. Rearrange things a bit to ensure that, whenever possible, the element is not shown until templates are rendered.

**Pull request merged**